includes/loadgraph.h

```diff
@@ -39,37 +33,20 @@ typedef enum {
     LG_COLOR_RED   = 0xB04F4F
 } LoadGraphColor;
 
-struct _LoadGraph {
-    GdkPixmap     *buf;
-    GtkWidget     *area;
-
-    GdkGC         *grid;
-    GdkGC         *trace;
-    GdkGC	  *fill;
-
-    gint     	  *data;
-    gfloat         scale;
+LoadGraph   *load_graph_new(gint size);
+void         load_graph_destroy(LoadGraph *lg);
+void         load_graph_configure_expose(LoadGraph *lg);
+GtkWidget   *load_graph_get_framed(LoadGraph *lg);
 
-    gint	   size;
-    gint	   width, height;
-    LoadGraphColor color;    
-    
-    gint	   max_value, remax_count;
-    
-    PangoLayout   *layout;
-    gchar	  *suffix;
-};
+void         load_graph_update(LoadGraph *lg, gdouble value);
+void         load_graph_update_ex(LoadGraph *lg, guint line, gdouble value);
```

Comment:
There are very few places where `load_graph_update()` is called. Maybe it would be better to make `load_graph_update_ex()` the default and only behavior to update the load graph.